README.md

```diff
@@ -1148,6 +1148,7 @@ Update Time, five active automations, webhooks.
 
 ## Low-code Platform
 
+  * [Basedash](https://www.basedash.com/?ref=free-for-dev) — Low-code platform for building internal admin panels and dashboards. Supports SQL databases and REST APIs.
```

Comment:
Thanks.  Please remove the tracking component of the link.

Reply:
Done!